MIGRATION.md

```diff
@@ -0,0 +1,87 @@
+# Migrating to the `cloudant-python-sdk` library 
+This document is to assist in migrating from the legacy `python-cloudant` to the newly supported `cloudant-python-sdk`.
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-python-sdk`:
+1. [Enviornment variables](https://github.com/IBM/cloudant-python-sdk#authenticate-with-environment-variables)
```

Comment:
Since then the link changed:
```suggestion
1. [Enviornment variables](https://github.com/IBM/cloudant-python-sdk#authentication-with-environment-variables)
```